Makefile

```diff
@@ -10,15 +12,12 @@ endif
 # append commit-sha to next version
 BUILD_VERSION := $(VERSION)
 ifeq ($(BUILD_VERSION),next)
+	CI_COMMIT_SHA ?= $(shell git rev-parse HEAD)
```

Comment:
Isn't `$CI_COMMIT_SHA` set by the CI already?

Reply:
yes but you can run Makefile outside of ci too - and there it's not